### ISSUE TITLE
FN S3261: No issue is raised when in file scoped namespace.

### ISSUE THREAD

**Issue by mary-georgiou-sonarsource** [SonarSource/sonar-dotnet#5979]
### Description

S3261 should raise an issue when a file scoped namespace is empty.

### Repro steps

```csharp
namespace Tests.Diagnostics; // FN
using System;
using System.Diagnostics;
```

### Related information

* C#/VB.NET Plugins version: 8.43.0.51858
